scalajvm/test/ApproverModelSpec.scala

```diff
@@ -1,23 +0,0 @@
-import org.specs2.mutable._
```

Comment:
And why are you removing the tests? The point is that you should add new and not remove existing :(